src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt

```diff
@@ -157,6 +157,18 @@ class StudyService(
         private val GET_ORGANIZATION_ID = """
             SELECT ${ORGANIZATION_ID.name} FROM ${ORGANIZATION_STUDIES.name} WHERE ${STUDY_ID.name} = ? LIMIT 1 
         """.trimIndent()
+
+        // How to only get studies that also the current user has READ permissions on?
```

Comment:
You can inner join with permission table ON study_id = acl_key[1]

Reply:
You could also filter by checking authorization on each study but that will be super slow for large number of studies. 

The other way is to invert question and ask which studies that the current user have read access on and the call get studies for those (iirc there's already a getAccessibleObjects(...) that takes securable object type as one of the parameters)

Reply:
updated sql to join with permissions table